content/2016.09.jedit-trackers-with-kinto-en.md

```diff
@@ -0,0 +1,403 @@
+Title: Getting jEdit Trackers off the Ground using Kinto and Kinto.js
```

Comment:
There's also a weak pun with crop dusting, which can be used to spread pesticides and thus kill "bugs". Or you could talk about a "pilot program", since a pilot can refer to the person flying an airplane.